int/api/webcreator.go

```diff
@@ -102,6 +115,18 @@ func UploadWebsiteHandler(params operations.WebsiteCreatorUploadParams) middlewa
 			})
 	}
 
+	contentType := http.DetectContentType(archive)
+
+	if contentType != "application/zip" {
```

Comment:
As this part of code is the same as the one between `l 46-57`
It may be a good idea to separate it in an other function ?

Reply:
I agree, it is done